pkg/kubelet/status/status_manager.go

```diff
@@ -329,6 +329,18 @@ func (m *manager) TerminatePod(pod *v1.Pod) {
 		oldStatus = &cachedStatus.status
 	}
 	status := *oldStatus.DeepCopy()
+	for i := range status.EphemeralContainerStatuses {
```

Comment:
can we block this section by validating the feature gate is enabled as the feature is still alpha?

Reply:
Done, @derekwaynecarr